### DISCUSSION TITLE
No mount location

### DISCUSSION THREAD

**Discussion by abror033** [iterate-ch/cyberduck#16519] [Q&A]
After Mac OS update to (15.0.1) my mountain duck (v 14.6.3)  replaced mount folder (Volume.noindex) to alias and while mounting the new connection the corresponding folder is not available. Because of that my local macros does not work since the path is not findable. 
Can someone help?

**Comment by tatze96** [comment]
We had to change the default mount location to a different one due to thumbnail display issues. You can set the mount location back to the previous default mount location within the Mountain Duck preferences *Sync* tab. Note that you may run into thumbnail display issues afterward.

Ensure to click on *Choose...* instead of selecting an entry from the dropdown menu to change the mount location successfully.

The previous default mount location was set to
`~/Library/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/Volumes.noindex`

**Reply by abror033** [reply]
Hi, and thanks for reply. 
As you can see from the screenshot, I can't select the mount location (shaded).

<img width="774" alt="Screenshot 2024-11-14 at 11 04 39" src="https://github.com/user-attachments/assets/61cc08cb-00b0-4ae1-bccb-6a78347628c4">

**Reply by avokadosauce** [reply]
Is there a fix for this? having the same problem.

**Reply by dkocher** [reply]
Running Mountain Duck installed from the Mac App Store does not allow choosing a custom mount location.
